master/internal/resourcemanagers/kubernetes_resource_manager.go

```diff
@@ -374,10 +374,6 @@ func (k *kubernetesResourceManager) assignResources(
 		//	k.addrToContainerID[req.TaskActor] = []cproto.ID{}
 		//}
 		k.addrToContainerID[req.TaskActor] = container.id
-		ctx.Tell(k.agent.handler, kubernetes.SetPodOrder{
```

Comment:
is this intentional?

Reply:
yes - the pod label doesn't allow the "-" character, so this code is pointless.